index.html

```diff
@@ -1,288 +1,272 @@
-          <div class="project-text">
-            <h2>
-              I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-            </h2>
-          </div>
-        </section>
-        <section class="contact-form">
-          <form action="https://formspree.io/f/xoqzrebz" method="post">
-            <input required type="text" name="name" placeholder="Full name" maxlength="30" pattern="\w{3,30}">
-            <input required type="email" name="email" id="email" placeholder="Email address" pattern="[a-z0-9._+-]+@[a-z0-9._+-]+\.[a-z]{2,}" >
-            <textarea required name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Hey I'd like to connect to..."></textarea>
-            <button class="button" type="submit"> Get in touch </button>
-          </form>
-
-          <div class="headline-paragraph">
-            <p>mutoni.cedric@gmail.com</p>
+      </div>
+      <div class="language-grid">
+        <div class="language-icon">
+          <img src="./assets/icon-skills.svg" alt="language icon">
+          <div class="language-title">
+            <h2>Skills</h2>
           </div>
-          <div class="headline-social bottom-social">
+          <div class="language-skills">
             <ul>
-              <li><a href="https://twitter.com/"><img src="./assets/twitter.svg" alt="twitter icon"/></a></li>
-              <li><a href="https://linkedin.com/"><img src="./assets/Linkedin.svg" alt="Linkedin icon"/></a></li>
-              <li><a href="https://medium.com/"><img src="./assets/medium.svg" alt="Medium icon"/></a></li>
-              <li><a href="https://github.com/"><img src="./assets/github.svg" alt="GitHub icon"/></a></li>
-              <li><a href="https://www.angellist.com/"><img src="./assets/angelist.svg" alt="Angelist icon"/></a></li>
+              <li>Database Management</li>
+              <li>Version Control</li>
+              <li>CLI</li>
+              <li>Web Development</li>
+              <li>API Design</li>
             </ul>
           </div>
-        </section>
+        </div>
+      </div>
+    </section>
+    <section class="text-form">
+      <div class="project-text">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+      </div>
+      <section class="contact-form">
+        <form action="https://formspree.io/f/xoqzrebz" method="post">
+          <input required type="text" name="name" placeholder="Full name" maxlength="30" pattern="\w{3,30}">
+          <input required type="email" name="email" id="email" placeholder="Email address" pattern="[a-z0-9._+-]+@[a-z0-9._+-]+\.[a-z]{2,}" >
+          <textarea required name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Hey I'd like to connect to..."></textarea>
+          <button class="button last-button" type="submit"> Get in touch </button>
+        </form>
```

Comment:
- [ ] Great job💪 on making your project look like the Figma design! 🙆‍♀️However, for the contact 📞section, there is still room for improvement. 🙏Please adjust the style of both the text and form elements, including the button text colors. This will help make your project look even more like the Figma design and achieve perfection

Your design:
![image](https://user-images.githubusercontent.com/63445083/236554007-7970522b-1229-4a52-9683-2a4dba3c9e0b.png)

Figma design:
![image](https://user-images.githubusercontent.com/63445083/236554247-9859fef8-99f2-4c28-aec0-1a331f90a70c.png)